src/Controller/Component/FlashComponent.php

```diff
@@ -111,6 +112,14 @@ public function set($message, array $options = [])
             $messages = (array)$this->_session->read('Flash.' . $options['key']);
         }
 
+        if ($options['duplicate'] === false) {
```

Comment:
There is a `$options['clear'] === false` above, 
was trying to make it all more same-like, thoughts?